packages/flutter_test/lib/src/test_pointer.dart

```diff
@@ -437,3 +437,20 @@ class TestGesture {
     });
   }
 }
+
+/// A pack of input PointerEvent queue.
+///
+/// [timeStamp] is used to indicate the time when the pack is received.
+///
+/// This is a simulation of how the framework is receiving input events from
+/// the engine. See [GestureBinding] and [PointerDataPacket].
+class PointerEventPack {
+  /// Creates a pack of PointerEvents.
+  PointerEventPack(this.timeStamp, this.events);
+
+  /// The time stamp of when the event happens
```

Comment:
I think this is the time stamp when the engine gives the whole packet to the framework? The time stamp of when the event happens should be `events[i].timeStamp`?